### ISSUE TITLE
False positives for TrailingComma cop, %w() arrays

### ISSUE THREAD

**Issue by mockdeep** [rubocop/rubocop#785]
When I have the following configuration:

``` ruby
TrailingComma:
  EnforcedStyleForMultiline: comma
```

I get false positives on multiline %w() arrays:

``` ruby
ingredients = %w(
  sausage
  anchovies
  olives
)
```
